src/main/java/com/atherys/quests/command/quest/AttachQuestCommand.java

```diff
@@ -0,0 +1,25 @@
+package com.atherys.quests.command.quest;
+
+import com.atherys.core.command.annotation.Aliases;
+import com.atherys.core.command.annotation.Children;
+import com.atherys.core.command.annotation.Description;
+import com.atherys.core.command.annotation.Permission;
+import org.spongepowered.api.command.CommandException;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.spec.CommandExecutor;
+import org.spongepowered.api.entity.living.player.Player;
+
+@Permission("atherysquests.admin.quest.attach")
+@Aliases("attach")
+@Description("Base command for attaching quests.")
+@Children({AttachQuestToLocationCommand.class, AttachQuestToItemCommand.class})
+public class AttachQuestCommand implements CommandExecutor {

```

Comment:
Where is the implementation for this?

Reply:
Aha, thanks. I need to get some pagination lists going for all of the base commands.